Utilities/Utilities/_Utils/_FeatureFlags/FeatureService.swift

```diff
@@ -7,15 +7,24 @@
 //
 
 import Foundation
+import Combine
 
 public protocol FeatureFlagsProtocol {
-    var featureFlags: [String: Any]? { get }
```

Comment:
The statsig SDK does not expose a list of feature flags, so this pattern had to change